tests/test_phase_psf.py

```diff
@@ -52,8 +53,51 @@ def test_aperture():
 
 
 @timer
+def test_atm_screen_size():
+    """Test for consistent AtmosphericScreen size and scale."""
+    screen_size = 10.0
+    screen_scale = 0.1
+    atm = galsim.AtmosphericScreen(screen_size=screen_size, screen_scale=screen_scale)
+    # AtmosphericScreen will preserve screen_scale, but will adjust screen_size as necessary to get
+    # a good FFT size.
+    assert atm.screen_scale == screen_scale
+    assert atm.screen_size != screen_size
+    np.testing.assert_equal(atm.screen_size, atm.npix * atm.screen_scale,
+                            "Inconsistent atmospheric screen size and scale.")
+
+
+@timer
+def test_structure_function():
+    """Test that AtmosphericScreen generates approximately the right structure function for infinite
+    outer scale.
+    """
+    rng = galsim.BaseDeviate(4815162342)
+    r0_500 = 0.2
+    L0 = None
+    screen_scale = 0.05
+    screen_size = 100.0
+
+    # Theoretical pure Kolmogorov structure function (at 500 nm!):
+    D_kolm = lambda r: 6.8839 * (r/r0_500)**(5./3)
+
+    atm = galsim.AtmosphericScreen(screen_size=screen_size, screen_scale=screen_scale,
+                                   r0_500=r0_500, L0=L0, rng=rng)
+    phase = atm.tab2d.table.getVals()[:-1, :-1].copy()
+    phase *= 2 * np.pi / 500.0  # nm -> radians
+    im = galsim.Image(phase, scale=screen_scale)
+    D_sim = galsim.utilities.structure_function(im)
+
+    for r in [0.5, 2.0, 5.0]:  # Only check values far from the screen size and scale.
+        # We're only attempting to verify that we haven't missed a factor of 2 or pi or
+        # something like that here, so set the rtol below to be *very* forgiving.  Since the
+        # structure function varies quite quickly as r**(5./3), this is still a useful test.
+        np.testing.assert_allclose(D_kolm(r), D_sim(r), rtol=0.5,
```

Comment:
Sure.  Here's the output of the print:

```
r   D_kolm   D_sim
0.5 31.7006305419 36.7609791124
2.0 319.522333717 322.01840719
5.0 1471.41292742 1274.34747692
```

So for these settings and random seed, it's actually good to about 15%.  Unfortunately, I don't have a good feeling for how consistent we expect these numbers to be, since I think that probably is sensitive to resolution, edge effects, and it's also stochastic.  I did add to the comment above the test:

```
        # We're only attempting to verify that we haven't missed a factor of 2 or pi or
        # something like that here, so set the rtol below to be *very* forgiving.  Since the
        # structure function varies quite quickly as r**(5./3), this is still a useful test.
        # For the parameters above (including the random seed), D_kolm(r) and D_sim(r) are actually
        # consistent at about the 15% level in the test below.  It's difficult to predict how
        # consistent they *should* be though, since the simulated structure function estimate is
        # sensitive to resolution and edge effects, as well as the particular realization of the
        # field.
```